src/evaluation_system/tests/plugin_manager_test.py

```diff
@@ -252,7 +252,10 @@ def runTool(self, config_dict=None):
             pm.reload_plugins()
             log.debug("post-loading: %s", list(pm.get_plugins()))
             assert "testplugin1" in list(pm.get_plugins())
-            assert "testplugin2" in list(pm.get_plugins())
+            if (
+                not in_wsl
+            ):  # WSL does not recognise $HOME as ~ apparently: skip this assert
+                assert "testplugin2" in list(pm.get_plugins())
 
```

Comment:
So what is done here, is testing the functionality of `os.path.expandvars` which shouldn't be really our concern here. Hence I think you don't have to add $HOME here, the `~` should also do it, or something else - it's not important. Also, while we're add it. Those tests are entirely designed for UNIX systems - think about all those forward slashes `/`. Maybe it's better to use pathlib for strings that should represent paths in the file system.

Reply:
in theory WSL is a subsystem that allows to run a Unix environment, the paths use forward slashes, and when testing jupyter within WSL  for `path = os.path.abspath(os.path.expandvars(os.path.expanduser(path)))` expandss `$HOME` correctly.

While when running the tests `~` works just fine, is `$HOME` the one that gives me the problems and I do not know why. 

But of course I can drop the second plugin, and remove what I added in the conftest.py